vehicle/autoware_raw_vehicle_cmd_converter/include/autoware_raw_vehicle_cmd_converter/node.hpp

```diff
@@ -102,16 +108,26 @@ class RawVehicleCommandConverterNode : public rclcpp::Node
   bool use_steer_ff_;
   bool use_steer_fb_;
   bool is_debugging_;
-  bool convert_accel_cmd_;  //!< @brief use accel or not
-  bool convert_brake_cmd_;  //!< @brief use brake or not
-  bool convert_steer_cmd_;  //!< @brief use steer or not
+  bool convert_accel_cmd_;                                             //!< @brief use accel or not
+  bool convert_brake_cmd_;                                             //!< @brief use brake or not
+  std::optional<std::string> convert_steer_cmd_method_{std::nullopt};  //!< @brief method to convert
+  double vgr_coef_a_{0.0};
+  double vgr_coef_b_{0.0};
+  double vgr_coef_c_{0.0};
```

Comment:
Why are these members necessary?

Reply:
not necessary. it was for before refactoring, so removed them.